test/controllers/integrations/buildkite_controller_test.rb

```diff
@@ -63,4 +63,29 @@
       response.status.must_equal 200
     end
   end
+
+  context 'when the buildkite_release_params hook gets trigger' do
+    before do
+      project.releases.destroy_all
+      project.builds.destroy_all
```

Comment:
1: We've tried the assert_difference, and it turn out without deleting the release, the test will hit the (:get, "https://api.github.com/repos/bar/foo/compare/abc...dc395381e650f3bac18457909880829fc20e34ba"). 

because it's trying to get the diff between previous release, so we have to stub it..

So I think it's easier just to delete here? 

2: Also I was trying to copy the example from base_controller_test.rb, in which it's doing the destoy_all thing